modules/core/shared/src/main/scala/gem/config/DynamicConfig.scala

```diff
@@ -186,4 +191,34 @@ object DynamicConfig {
       GmosSouth(GmosCommonDynamicConfig.Default, None, None, None)
   }
 
+  import GnirsConfig.GnirsFpu
+
+  final case class Gnirs(
+    camera:               GnirsCamera,
+    decker:               GnirsDecker,
+    disperser:            GnirsDisperser,
+    disperserOrder:       GnirsDisperserOrder,
+    exposureTime:         Duration,
+    filter:               GnirsFilter,
+    fpu:                  GnirsFpu,
+    prism:                GnirsPrism,
+    readMode:             GnirsReadMode,
+    wavelengthSuggestion: GnirsWavelengthSuggestion
+  ) extends DynamicConfig.Impl(Instrument.Gnirs)
+
+  object Gnirs {
+    val Default: Gnirs = Gnirs(
+      GnirsCamera.ShortBlue,
+      GnirsDecker.Acquisition,
+      GnirsDisperser.D32,
+      GnirsDisperserOrder.Three,
+      java.time.Duration.ZERO,
```

Comment:
The default exposure time for this configuration according to the OT is 17 seconds.